mithril-stm/README.md

```diff
@@ -91,50 +124,56 @@ fn main() {
     // Aggregate with random parties
     let msig = clerk.aggregate(&sigs, &msg);
 
-    assert!(msig.is_ok(), "aggregation failed");
-    assert!(msig.unwrap().verify(&msg, &clerk.compute_avk(), &params).is_ok());
+    match msig {
+        Ok(aggr) => {
+            println!("Aggregate ok");
+            assert!(aggr.verify(&msg, &clerk.compute_avk(), &params).is_ok());
+        }
+        Err(AggregationError::NotEnoughSignatures(n, k)) => {
+            println!("Not enough signatures");
+            assert!(n < params.k && k == params.k)
+        }
+        Err(AggregationError::UsizeConversionInvalid) => {
+            println!("Invalid usize conversion");
+        }
+    }
 }
 ```
 
-# Test and Benchmarks
-You can run tests of the library using `cargo test` (we recommend to use the `--release` flag, otherwise
-the tests might take a while) and run benchmarks using `cargo bench`. This crate uses `criterion` to run
-benchmarks.
+## Benchmarks
 
-We have run the benchmarks on an Apple M1 Pro machine with 16 GB of RAM, on macOS 12.6.
+Here we give the benchmark results of STM for size and time. We run the benchmarks on macOS 12.6 on an Apple M1 Pro machine with 16 GB of RAM.
 
-Note that single signatures in batch compat version does not depend on any variable and size of an individual signature is `176` bytes.
+Note that single signatures in batch compatible version do not depend on any variable and **the size of an individual signature is 176 bytes**.
```

Comment:
The constant size of individual signatures is not tied to the batch proofs. Instead, it is tied to the fact that we simply do not include merkle paths in individual signatures, I would suggest to change this sentence as: 

`Note that single signatures do not contain the Merkle path, and therefore have a **constant size of 176 bytes**.`

Reply:
Actually, signatures don't have constant size, right? They depend on how many indices they've won. @curiecrypt